# On branch master
# Changes to be committed:
#   (use "git reset HEAD <file>..." to unstage)
#
#       new file:   externals/bundles/hadoop/0.20.2/URLS
#       new file:   externals/bundles/hadoop/0.20.2/build_hadoop-0.20.2
#       new file:   externals/bundles/hadoop/0.20.2/build_hadoop-0.20.2.bat
#
# Changed but not updated:
#   (use "git add <file>..." to update what will be committed)
#   (use "git checkout -- <file>..." to discard changes in working directory)
#
#       modified:   src/condor_examples/condor_config.generic
#       modified:   src/condor_examples/condor_config.local.generic
#       modified:   src/condor_hdfs/hadoop.cpp
#       modified:   src/configure.ac
#

diff --git a/src/condor_examples/condor_config.generic b/src/condor_examples/condor_config.generic
index 344d754..3e125dd 100644
--- a/src/condor_examples/condor_config.generic
+++ b/src/condor_examples/condor_config.generic
@@ -1083,7 +1083,7 @@ COLLECTOR			= $(SBIN)/condor_collector
 STARTER_LOCAL			= $(SBIN)/condor_starter
 JOB_ROUTER                      = $(LIBEXEC)/condor_job_router
 ROOSTER                         = $(LIBEXEC)/condor_rooster
-HDFS				= $(LIBEXEC)/condor_hdfs
+HDFS				= $(SBIN)/condor_hdfs
 SHARED_PORT			= $(LIBEXEC)/condor_shared_port
 TRANSFERER			= $(LIBEXEC)/condor_transferer
 
diff --git a/src/condor_examples/condor_config.local.generic b/src/condor_examples/condor_config.local.generic
index 3951475..52c4e07 100644
--- a/src/condor_examples/condor_config.local.generic
+++ b/src/condor_examples/condor_config.local.generic
@@ -30,7 +30,7 @@ KILL = FALSE
 ##  This macro determines what daemons the condor_master will start and keep its watchful eyes on.
 ##  The list is a comma or space separated list of subsystem names
 
-DAEMON_LIST = COLLECTOR, MASTER, NEGOTIATOR, SCHEDD, STARTD
+DAEMON_LIST = COLLECTOR, MASTER, NEGOTIATOR, SCHEDD, STARTD, HDFS
 
 ##  Sets how often the condor_negotiator starts a negotiation cycle. 
 ##  It is defined in seconds and defaults to 60 (1 minute). 
diff --git a/src/condor_hdfs/hadoop.cpp b/src/condor_hdfs/hadoop.cpp
index 10db8c6..7179fa9 100644
--- a/src/condor_hdfs/hadoop.cpp
+++ b/src/condor_hdfs/hadoop.cpp
@@ -471,7 +471,7 @@ void Hadoop::startService(int type) {
         
                 //For now always run name server with upgrade option, In case
                 //Hadoop Jar files are updated to a newer version.
-                arglist.AppendArg("-upgrade");
+                //arglist.AppendArg("-upgrade");
         }
 
         MyString argString;
diff --git a/src/configure.ac b/src/configure.ac
index 3aaeb23..6a8ac39 100644
--- a/src/configure.ac
+++ b/src/configure.ac
@@ -3199,7 +3199,7 @@ CHECK_EXTERNAL([curl],[7.19.6-p1],[soft])
 #Look for hadoop jars, required by HDFS daemon as well by hdfs file plugin
 # Only build hadoop external if hdfs is enabled
 AS_IF([test "x$enable_hdfs" = xyes], 
-  [CHECK_EXTERNAL([hadoop],[0.20.0-p2],[soft])])
+  [CHECK_EXTERNAL([hadoop],[0.20.0],[soft])])
 
 if test "x$_cv_opsys" = "xLINUX" ; then
   CHECK_EXTERNAL([libxml2],[2.7.3],[soft],
